db_client.py

```diff
@@ -0,0 +1,80 @@
+import pymongo
+from item import Item
+from status import NOT_STARTED
+from datetime import datetime
+from bson.objectid import ObjectId
+from dataclasses import dataclass
+
+DATE_TIME_FORMAT = "%Y-%m-%dT%H:%M:%S.%fZ"
+
+
+@dataclass
+class DbConfig:
+    db_name: str
+    db_user: str
+    db_pass: str
+
+
+class DbClient:
+    def __init__(self, db_config: DbConfig, overwrite_db_name=False):
+        db_name = "test" if overwrite_db_name else db_config.db_name
+        client = pymongo.MongoClient(
+            f"mongodb+srv://{db_config.db_user}:{db_config.db_pass}@cluster0.bgo8m.mongodb.net/{db_name}?retryWrites=true&w=majority"
```

Comment:
Best not to hardcode the database URL (and you'll need to change it later exercises)